packages/plugins/Solana/src/storage/index.ts

```diff
@@ -14,3 +15,15 @@ export function setupStorage(_: typeof storage) {
 export function getStorage() {
     return storage.storage
 }
+
+type BNLike = {
+    _bn: string
+}
+export async function storeConnection(pubKey: string | BNLike, chainId?: number) {
```

Comment:
Solana should have an enumerate value for chain IDs. Therefore, you could change the `number` type into `ChainId`.

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/5921/commits/e6987f670b158fa0f4b121000a6fba537bbda906